src/lib.rs

```diff
@@ -2855,6 +2865,7 @@ shims! [
     (asterisk, Token::into_asterisk, Error::ExpectedAsterisk),
     (at, Token::into_at, Error::ExpectedAt),
     (bang, Token::into_bang, Error::ExpectedBang),
+    (box_pattern, Token::into_box, Error::ExpectedBox),
```

Comment:
You can just reuse `kw_box`, defined above (`kw_` is to avoid conflict with the *real* keyword) :-)

Reply:
Ah dang, i knew it would be there but must have missed it! :D